src/OptionsResolver.php

```diff
@@ -113,7 +113,7 @@ public function getDefault($option)
      */
     public function getDefaults()
     {
-        return $this->defaults;
+        return $this->defaults; // maybe we get to SF to break open the parent?
     }
 
```

Comment:
I noticed that a few properties in this class are the same as the parents.

For the `normalizers` I wonder if this one can be removed add done through the parent completely.

For the `defaults` I see you need it as the parent doesn't allow to get the `defaults`. I wonder if maybe this can be changed in the SF class itself. However that is of course out of scope. I just forgot to remove the comment.